src/pages/components/progress-bar/usage.mdx

```diff
@@ -0,0 +1,504 @@
+information about the progression in the optional helper text.
+
+### Anatomy
+
+<Row>
+<Column colLg={8}>
+
+![Progress bar anatomy](images/progress-bar-usage-6.png)
+
+</Column>
+</Row>
+
+1. **Label:** The text describes the process that the progress bar refers to.
+   Label can be visually hidden, but requires to be defined to be accessible.
+2. **Track:** The static area that the bar indicator moves on top of and acts as
+   a fixed visual reference of what the total length and duration of the process
+   could be.
+3. **Bar indicator:** Indicates how much the process has progressed.
+4. **Helper text (optional):** Assistive text to show additional information
+   about the process taking place. For determinate, the information is usually
+   values of number or percentage. Sometimes, a fraction or ratio may be shown.
+   These values are not applicable when a progress bar is indeterminate.
+
+### Sizing
+
+#### Height
+
+The progress bar is offered in two different sizes—default (8px) and small
+(4px). The default progress bar height is typically used when there is large
+amounts of space on a page. The small progress bar height is commonly used when
+space is restricted and can be placed within cards, data table, or side panels.
+
+<Row>
+<Column colLg={8}>
+
+![Progress bar sizes](images/progress-bar-usage-7.png)
+
+</Column>
+</Row>
+
+#### Width
+
+The width of a progress bar can be customized appropriately for its context. The
+minimum width of a progress bar is 48px and keep its width to a maximum of six
+columns when possible. If the progress bar width is too long it can reduce
+readability. Don't increase the progress bar length to fill the entire width of
+a window or application.
+
+<DoDontRow>
+
+  <DoDont caption="Do keep a width within six columns.">
+
+![Do keep a width within six columns.](images/progress-bar-usage-8.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Don't exceed a width of six columns.">
+
+![Don't exceed a width of six columns.](images/progress-bar-usage-9.png)
+
+  </DoDont>
+</DoDontRow>
+
+### Alignment
+
+Text alignment depends on the context and area given. As long as the text is
+placed close to the progress bar, it’s up to designers to provide the best
+judgement that fits the context.
+
+<Row>
+<Column colLg={12}>
+
+![Progress bar alignments](images/progress-bar-usage-10.png)
+
+<Caption>A few common examples of text alignmnet for progress bars</Caption>
+
+</Column>
+</Row>
+
+#### Internationalization (RTL)
+
+For RTL (right-to-left) languages, the layout of the progress bar is mirrored
+for both determinate and indeterminate options. The label is right-aligned, the
+value is left-aligned, and the fill progresses from right to left. Keep in mind
+that the placement of the percent sign differs depending on the locale.
+
+<Row>
+<Column colLg={6}>
+
+![Progress bar RTL](images/progress-bar-usage-11.gif)
+
+</Column>
+</Row>
+
+#### Text placement
```

Comment:
I would move this section above section "Internationalization" section because it related more to that paragraph under alignment. They seem to cover similar related topics